sources/platform/actors/development/actor_definition/source_code.md

```diff
@@ -37,11 +39,20 @@ COPY . ./
 CMD npm start --silent
 ```
 
-will build the Actor from the `apify/actor-node:16` image, copy the `package.json` and `package-lock.json` files to the image.
+1. Builds the Actor from the `apify/actor-node:16` base image.
+2. Copies the `package.json` and `package-lock.json` files to the image.
+3. Installs the `npm` packages specified in `packages.json`, omitting development and optional dependencies.
+4. Copies the rest of the source code to the image
+5. Runs the `npm start` command defined in `package.json`
```

Comment:
Question: Maybe it would make more sense to inline this as comments to the Dockerfile?

Reply:
I've slightly restructured the whole doc around this part by:

First providing the whole code block and only after that breaking it up with explanations, @valekjo please let me know if this works better

Reply:
Nice! Thanks.